test/extended/networking/ipsec.go

```diff
@@ -455,9 +482,50 @@ var _ = g.Describe("[sig-network][Feature:IPsec]", g.Ordered, func() {
 				}
 			}
 			o.Expect(config.dstNodeConfig.nodeIP).NotTo(o.BeEmpty())
+
+			g.By("configure IPsec certs on the worker nodes")
+			// The certificates in the Machine Config has validity period of 120 months starting from April 11, 2024.
+			// so proceed with test if system date is before April 10, 2034. Otherwise fail the test.
+			if !time.Now().Before(certExpirationDate) {
+				framework.Failf("certficates in the Machine Config are expired, Please consider recreating those certificates")
+			}
+			nsCertMachineConfig, err := createIPsecCertsMachineConfig(oc)
```

Comment:
I guess this does not cause a reboot because they are just ceerts

Reply:
this still causes reboot of worker nodes and certs are deployed via machine config.